recipes/ARIA-tools/meta.yaml

```diff
@@ -0,0 +1,87 @@
+{% set name = "ARIA-tools" %}
+{% set version = "1.1.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/aria-tools/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: c544fee1c539f38e23897d27332a6af0f45d48874e651a096993dbd311cbbf48
+
+build:
+  script:
+    - export SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+    - {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [py<38 or win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - python
+    - pip
+    - pybind11 >=2.5.0
+    - setuptools >61.0
+    - setuptools_scm >6.2
+    - python-build
+    - numpy
+    - cython
+  run:
+    - python
+    - asf_search
+    - cartopy
+    - gdal >=3.4.1
+    - h5py
+    - joblib
+    - matplotlib-base
+    - netcdf4
+    - pandas
+    - xarray
+    - rasterio
+    - rioxarray
+    - pip
+    - pyproj
+    - requests
+    - scipy <1.10
+    - mintpy
+    - shapely
+
+test:
+  imports:
+    - ARIAtools
+  requires:
+    - pip
+  commands:
+    - pip check
+    # - ariaExtract.py -h
+    # - ariaDownload.py -h
+    # - ariaTSsetup.py -h
+    # - ariaPlot -h
+    # - ariaMisclosure.py -h
+    # - ariaAOIassist.py -h
```

Comment:
Why are these commented out? If these are entry-points you need to declare them in the recipe like:

https://github.com/conda-forge/ipython-feedstock/blob/55f53817eb6360ce6a588ae9d4d8a3653d069857/recipe/meta.yaml#L24-L26

Reply:
These commands all require earthdata login credentials and we are not going to set that up upon this initial release - I included these commands under `entry_points` as you indicated. Thank you for this suggestion!

Reply:
I had to comment out the entrypoints I suggested because ARIA-tools currently using the `scripts` paradigm with a setup.py. I created an issue ticket to fix this down the line. Hoping this can be released via conda-forge and fixed down  the road.

https://github.com/aria-tools/ARIA-tools/issues/365